ingestion/src/metadata/ingestion/ometa/utils.py

```diff
@@ -66,3 +67,17 @@ def uuid_to_str(entity_id: Union[str, basic.Uuid]) -> str:
         return str(entity_id.__root__)
 
     return entity_id
+
+
+def fqdn_to_str(fqdn: Union[str, FullyQualifiedEntityName]) -> str:
```

Comment:
Here I followed the same approach as with UUID. However, if after EntityName PR we have a similar scenario, we can just prepare a generalized method to handle either str or models with __root__.

@ayush-shah feel free to give this a more elegant go

Reply:
I think we can merge 2 methods into one - i.e. UUID method uuid_to_str and fqdn_to_str can be merged using a Union for both models or 
list of models that need __root__

Reply:
cc: @pmbrull

Reply:
ok got a stab at it as I also needed to fix the EntityName Thanks